### PR TITLE
Windows UWP Support 

### PR DESCRIPTION
**Pull request by lznt**
Hello,

We are developing our app to support: https://github.com/ReactWindows/react-native-windows and are using this library on our Android and iOS code. 

Here is a quick expand to support uwp react native apps. It works as a reference library in any VS Solution project. Works with the same js bridge file as the Android and iOS.

If you need any more info im happy to provide.

BR,
Lasse

### CODE REVIEW

**Comment by GantMan** [comment]
looks awesome.  I have no way of testing this, as I don't have Studio on my one Windows machine.  Anyone else?

**Comment by lznt** [comment]
Made some improvements.

**Comment by GantMan** [comment]
can anyone validate this works?  I'll be starting windows in dev in about a week, I guess I can validate it then.

**Comment by josephan** [comment]
I have tried using this PR but I failed. Here were my steps:
- run `npm uninstall --save react-native-device-info`
- run  `npm install --save lznt/react-native-device-info`
- verify that the `windows` directory is in `./node_modules/react-native-device-info`
- run `react-native run-windows`
- get the same error as I had from master -v 0.9.3

Another difficulty is that there are no docs for installation and I wasn't sure what to do as there weren't any solutions file to reference.

**Comment by josephan** [comment]
it would also be nice to add a `.gitignore` like https://github.com/EstebanFuentealba/react-native-share/blob/master/windows/.gitignore

File: windows/DeviceInfo/project.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿{
+  "dependencies": {
+    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
+    "Newtonsoft.Json": "9.0.1"
```
Comment (josephan) [inline windows/DeviceInfo/project.json]: could you clarify why this dependency was added? I don't see it used.

**Comment by rmevans9** [comment]
@josephan I might be wrong but I think you need to add the RNDevicePackage to the MainPage.cs NOT the Modules.

**Comment by machour** [comment]
Hi there!

I'm a new maintainer on this project and I'm trying to clear the PR queue.
Are you still available to bring this PR up to date with latest changes?

Please let me know, thank you!

**Comment by machour** [comment]
Closing this stale PR.

**Pull request closed without merging**